system/jlib/jexcept.cpp

```diff
@@ -1050,7 +1050,7 @@ void excsighandler(int signum, siginfo_t *info, void *extra)
     PROGLOG("================================================");
     PROGLOG("Signal:    %d %s",signum,strsignal(signum));
     PROGLOG("Fault IP:  %08X", ip);
-    PROGLOG("Accessing: %08X", info->si_addr);
+    PROGLOG("Accessing: %08X", (unsigned) info->si_addr);
```

Comment:
It's in a 32-bit section, as per the #ifdef above. %p would also have worked, I was just going down the "mimimum changes in obsolescent code" route...